src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/ConstructorWriter.java

```diff
@@ -70,6 +70,14 @@
      */
     void addDeprecated(ExecutableElement constructor, Content constructorDocTree);
 
+    /**
+     * Add the preview output for the given member.
+     *
+     * @param member the member being documented
+     * @param annotationDocTree content tree to which the preview information will be added
+     */
+    void addPreview(ExecutableElement member, Content contentTree);
```

Comment:
Note to javadoc-team:  Consider using default methods to provide an empty implementation.